lib/uploadcare/api/file_storage_api.rb

```diff
@@ -0,0 +1,29 @@
+module Uploadcare
+  module FileStorageApi
+    BATCH_SIZE = 100
+
+    def store_files(objects)
+      in_batches(objects) { |batch| put "/files/storage/", to_uuids(batch) }
+    end
+
+    def delete_files(objects)
+      in_batches(objects) { |batch| delete "/files/storage/", to_uuids(batch) }
+    end
+
+    private
+
+    def in_batches(enum)
+      enum.each_slice(BATCH_SIZE) { |batch| yield batch }
+    end
+
+    def to_uuids(objects)
+      objects.map do |object|
+        case object
+        when Uploadcare::Api::File then object.uuid
+        when /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/ then object
+        else raise(ArgumentError, "Unable to convert object to uuid: #{object}")
```

Comment:
The thing is that API accepts and ignores incorrect data and adds those in response.
I don't think it's good idea to make client more strict

```
{
  "status": "ok",
  "problems": {
    "21975c81-7f57-4c7a-aef9-acfe28779f78": "Missing in the project",
    "4j334o01-8bs3": "Invalid"
  },
  "result":[
    {
      "uuid": "cbaf2d73-5169-4b2b-a543-496cf2813dff",
      ...
    }
  ]
}
```